app/controllers/admin/users_controller.rb

```diff
@@ -104,14 +104,28 @@ def user_status
       begin
         Moderator::ManageActivityAndRoles.handle_user_roles(admin: current_user, user: @user, user_params: user_params)
         flash[:success] = I18n.t("admin.users_controller.updated")
+        respond_to do |format|
+          format.html do
+            handle_admin_user_path_redirect(params[:id])
+          end
+          format.json do
+            render json: { success: true, status: :ok }
```

Comment:
did you mean to have the http status be `:ok` here? if so, this should probably read:

`render json: { success: true }, status: :ok`